contents/customers/brainboard.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: How Brainboard raised conversion rates from 20% to 70% with PostHog
+customer: Brainboard
+logo: ../images/customers/brainboard/brainboard-logo.png
+featuredImage: ../images/customers/brainboard/featured.png
+industries:
+    - SaaS
+    - Developer tool
+users:
+    - Product
+    - Engineering
+    - Growth
+    - Marketing
+toolsUsed:
+    - Experimentation
+    - Self-hosted
+    - Insights
+    - Apps
+---
+
+[Brainboard](https://www.brainboard.co/) is a DevOps platform which enables cloud architects and engineers to visually design, deploy and manage cloud infrastructures collaboratively. Since joining the [Y Combinator](/customers/ycombinator) W22 batch Brainboard has helped users build over 75,000 architectures, and enabled thousands of platform engineers to cut delivery times by 50%.
+
+Typical for a YC start-up, Brainboard’s team had ambitious goals for what they wanted to achieve with their product data. As a result, the team considered many different tools before choosing PostHog. 
+
+“We tried Segment, Metabase, Mixpanel, Datadog and more,” says Stephane Boghossian, Brainboard’s Growth Architect. “But all of them were built for just a single type of users, like marketing or product.”
+
+“PostHog was the only which enabled everyone to both find insights and make data-driven changes,” agreed Jeremy Albinet, Brainboard’s CTO. “The marketing team can use PostHog for our website and landing pages, but my R&D team can manage the product too.”
+
+### Analytics at Brainboard: OKRs and experiments
+
+As infrastructure specialists, Brainboard unsurprisingly decided to self-host PostHog and manage its own deployment. Soon afterward, the team started building insights and centralizing  [OKRs](/blog/north-star-metrics) and other important metrics into company dashboards.
+
+“PostHog is our source of truth for everything,” says Stephane. “For example, we set up a [funnel](/product/funnels) which tracks from MQLs, to PQLs, to SQL, and so on. Number of website visits, sign-ups, breakdown per industry, errors, feedback…we track the entire user journey.”
```

Comment:
I don't think the “PostHog is our source of truth for everything,” quote should be here when the below section is all about it.